contrib/test.sh

```diff
@@ -56,7 +56,7 @@ done
 
 # Build the docs if told to (this only works with the nightly toolchain)
 if [ "$DO_DOCS" = true ]; then
-    RUSTDOCFLAGS="--cfg docsrs" cargo doc --all --features="$FEATURES"
+    RUSTDOCFLAGS="--cfg docsrs" cargo +nightly rustdoc --features="$FEATURES" -- -D rustdoc::broken-intra-doc-links
```

Comment:
Why `rustdoc` command instead of `doc`?

Reply:
Note for reviewers: this worked without `+nightly` not because we forgot to call it in CI but because we set nightly at the caller.

Reply:
When I tried with `doc` I get this error but `rustdoc` works.
```
$ RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --features="$FEATURES" -- -D rustdoc::broken-intra-doc-links
error: Found argument '-D' which wasn't expected, or isn't valid in this context

If you tried to supply `-D` as a value rather than a flag, use `-- -D
```

Seems the difference between `cargo doc` and `cargo rustdoc` is the addition of custom flags
```
$ cargo rustdoc -h
cargo-rustdoc
Build a package's documentation, using specified custom flags.
```

Reply:
Maybe RUSTDOCFLAGS="--cfg docsrs -D rustdoc::broken-intra-doc-links" would work, but it most likely doesn't matter.

Reply:
Will leave is is for now, thanks man.